### DISCUSSION TITLE
Add a slot to dropdown for bottom of the list (not the footer)

### DISCUSSION THREAD

**Discussion by stichingsd-vitrion** [primefaces/community#1144] [PrimeVue]
Hi there!  A slot for adding something at the end of the list or at bottom inside scroll container would be lovely

This way we can use https://github.com/oumoussa98/vue3-infinite-loading or any other scrolling in viewport detections to load next page.

Following up on this..
https://github.com/orgs/primefaces/discussions/1143

This would even be a better solution, but for now a slot would solve many issues, currently we wrap many of you components in our own wrappers to add missing or custom functionalities, and this is one we tought would be great for the community ^_^

**Comment by ipimpat** [comment]
The lack of slots that allow for the easy extension or replacement of built-in component logic is probably the most frustrating aspect of using PrimeVue.

**Reply by Acetyld** [reply]
Tbh for me is the only real missing thing

RadioButton should also have more slots to create stuff like https://x.com/artur_bien/status/1758887013283996007?s=46&t=rlOtJk0MpzWGl97M0-KQVA
So we can use the keyboard navigation, actually im gonna make a discussion for both things

**Reply by Acetyld** [reply]
@ipimpat https://github.com/orgs/primefaces/discussions/1217 posted 💪🏻 upvote would be nice 😉
